api/server/testrun.go

```diff
@@ -67,44 +69,34 @@ func (tapi ToDDApi) Run(w http.ResponseWriter, r *http.Request) {
 	if !testRunExists {
 		log.Warnf("Client requested run of testrun object, but %s was not found.", testRunInfo.TestRunName)
 		fmt.Fprint(w, "notfound")
-	} else {
-
-		// Populate sourceOverrideMap dict
-		sourceOverrideMap := map[string]string{
-			"SourceGroup": testRunInfo.SourceGroup,
-			"SourceApp":   testRunInfo.SourceApp,
-			"SourceArgs":  testRunInfo.SourceArgs,
-		}
+	}
 
-		// Send back the testrun UUID
-		testUUID := testrun.Start(tapi.cfg, finalObj.(objects.TestRunObject), sourceOverrideMap)
-		fmt.Fprint(w, testUUID)
+	// Populate sourceOverrideMap dict
+	sourceOverrideMap := map[string]string{
+		"SourceGroup": testRunInfo.SourceGroup,
+		"SourceApp":   testRunInfo.SourceApp,
+		"SourceArgs":  testRunInfo.SourceArgs,
 	}
+
+	// Send back the testrun UUID
+	testUUID := testrun.Start(tapi.cfg, finalObj.(objects.TestRunObject), sourceOverrideMap)
+	fmt.Fprint(w, testUUID)
 }
 
 // TestData will retrieve clean test data by test UUID
 func (tapi ToDDApi) TestData(w http.ResponseWriter, r *http.Request) {
-	// Retrieve query values
-	values := r.URL.Query()
-
-	var tdb = db.NewToddDB(tapi.cfg)
-
-	var testData string
-
 	// Make sure UUID string is provided
-	if testUuid, ok := values["testUuid"]; ok {
-
-		// Make sure UUID string actually contains something
-		if len(testUuid[0]) > 0 {
-			testData = tdb.DatabasePackage.GetCleanTestData(testUuid[0])
-		} else {
-			fmt.Fprint(w, "Error, test UUID not found.")
-		}
-
-	} else { // UUID not provided; get all agents
+	uuid := r.URL.Query().Get("uuid")
```

Comment:
Could you please leave this at ```testUuid``` for now? Or change the expected URL parameter in ```api/client/run.go```? This change broke the post-test analytics printout from the client